spec/request/api/v1/sessions/create_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe 'POST api/v1/users/sign_in', type: :request do
+
+  subject do
+    post new_user_session_path, params: params, as: :json
+    response
+  end
+
+  let(:password) { 'password' }
+  let(:token) do
+    {
+      '70crCAAYmNP1xLkKKM09zA' =>
+      {
+        'token' => '$2a$10$mSeRnpVMaaegCpn3AhORGe5wajFhgMoBjGIrMwq4Qq2mP6f/OHu1y',
+        'expiry' => 153_574_356_4
+      }
+    }
+  end
+
+  let(:user) { create(:user, password: password, tokens: token) }
+
+  context 'with correct params' do
+    let(:params) do
+    {
+      user: {
+        email: user.email,
+        password: password
+      }
+    }
+    end
+
+    it 'returns success' do
+        expect(subject).to have_http_status(:success)
```

Comment:
here you used 4 spaces indentation instead of 2.
Do you have `rubocop` configured?